clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/RecoveryIntegrationSpec.scala

```diff
@@ -233,7 +252,20 @@ class RecoveryIntegrationSpec
         // Already sorted by construction. Remove null and internal values here too.
```

Comment:
Can't comment on it directly, but instead of adding a separate loop check for the expected keys I'd just change `mapper` (above) to inject the default values. Then the existing `should contain` will cover the default-adding (and it'll give a better message than "false was not equal to true" when it fails).

Reply:
Since the mapper is used for the actual values as well, wouldn't that not be testing anything?

Reply:
Good point, it shouldn't go in `mapper`. I'd still say injecting the default fields into the expected and doing a single `contains` check is clearer than the double-check setup, and covers more (right now you're testing that the default keys get added to the JSON, but not that the keys have the expected values). I'd tack on a second `map` call in the `val sortedExpected =` statement.